src/desktop/apps/order2/server.js

```diff
@@ -51,6 +51,7 @@ app.get("/order2/:orderID*", async (req, res, next) => {
         headerLogoHref: res.locals.sd.REFERRER,
         options: {
           stripev3: true,
+          hideLogo: res.locals.sd.EIGEN,
```

Comment:
Since this isn't just hiding the logo but is also adding some behavior (top margin to accommodate the back button), can we rename `hideLogo` to something more specific to this use? (maybe `hideLogoForEigen`?).

#minor but it also might be cleaner to pull this variable outside of the `options` hash, like we do for `headerLogoHref`.